code/game/gamemodes/wizard/artefact.dm

```diff
@@ -213,3 +213,383 @@
+		for(var/F in faction)
+			if(F in user.faction)
+				faction_check = 1
+				break
+		if(faction_check == 0)
+			faction = list("[user.real_name]")
+			assigned = "[user.real_name]"
+			user.faction = list("[user.real_name]")
+			user << "You bind the sword to yourself. You can now use it to summon help."
+			if(!usr.mind.special_role)
+				if(prob(probability_evil))
+					user << "<span class='warning'><B>With your new found power you could easily conquer the station!</B></span>"
+					var/datum/objective/hijackclone/hijack_objective = new /datum/objective/hijackclone
+					hijack_objective.owner = usr.mind
+					usr.mind.objectives += hijack_objective
+					hijack_objective.explanation_text = "Ensure only [usr.real_name] and their copies are on the shuttle!"
+					usr << "<B>Objective #[1]</B>: [hijack_objective.explanation_text]"
+					ticker.mode.traitors += usr.mind
+					usr.mind.special_role = "[usr.real_name] Prime"
+					evil = TRUE
+				else
+					user << "<span class='warning'><B>With your new found power you could easily defend the station!</B></span>"
+					var/datum/objective/survive/new_objective = new /datum/objective/survive
+					new_objective.owner = usr.mind
+					new_objective.explanation_text = "Survive, and help defend the innocent from the mobs of multiverse clones."
+					usr << "<B>Objective #[1]</B>: [new_objective.explanation_text]"
+					usr.mind.objectives += new_objective
+					ticker.mode.traitors += usr.mind
+					usr.mind.special_role = "[usr.real_name] Prime"
+					evil = FALSE
+		else
+			var/list/candidates = get_candidates(BE_WIZARD)
+			if(candidates.len)
+				var/client/C = pick(candidates)
+				spawn_copy(C, get_turf(user.loc), user)
+				user << "<span class='warning'><B>The sword flashes, and you find yourself face to face with...you!</B></span>"
+				cooldown = world.time + cooldown_between_uses
+				for(var/obj/item/weapon/multisword/M in multiverse)
+					if(M.assigned == assigned)
+						M.cooldown = cooldown
+
+			else
+				user << "You fail to summon any copies of yourself. Perhaps you should try again in a bit."
+	else
+		user << "<span class='warning'><B>[src] is recharging! Keep in mind it shares a cooldown with the swords wielded by your copies.</span>"
+
+
+/obj/item/weapon/multisword/proc/spawn_copy(var/client/C, var/turf/T)
+	var/mob/living/carbon/human/M = new/mob/living/carbon/human(T)
+	if(duplicate_self)
+		usr.client.prefs.copy_to(M)
+	else
+		C.prefs.copy_to(M)
+	M.key = C.key
+	M.mind.name = usr.real_name
+	M << "<B>You are an alternate version of [usr.real_name] from another universe! Help them accomplish their goals at all costs.</B>"
+	M.faction = list("[usr.real_name]")
+	if(duplicate_self)
+		M.set_species(usr.get_species()) //duplicate the sword user's species.
+	else
+		if(prob(50))
+			var/list/all_species = list("Human","Unathi","Skrell","Tajaran","Kidan","Golem","Diona","Machine","Slime People","Grey","Vulpkanin")
+			M.set_species(pick(all_species))
```

Comment:
Well, it might have been fixed, can't remember- but set_species didn't properly null the hair style and facial hair style last time I checked